source/monitoring/installnewrelic.md

```diff
@@ -34,6 +34,16 @@ New Relic offers a [free tier account](https://newrelic.com/signup/) with "100 G
  $ newrelic-install install
  ```
 
+```eval_rst
```

Comment:
I would change this to not be a note, and instead be under a new subheading such as # Installing the agent on Plesk/cPanel 
Mention that Plesk and cPanel install PHP binaries to alternate locations, so you will need to use the following commands to ensure these are picked up. This will also apply to servers with non-default PHP installation paths.

Then, provide the example for Plesk also. If you check in the UKFast testing account, one of the servers is a Plesk test server,